src/style.css

```diff
@@ -1 +1,93 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc") format("truetype");
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.search-bar {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-bar--big{
+  width: 100%;
+  height: 70px;
+}
+
+.search-form--small{
+  width: 100%;
+  height: 42px;
+}
```

Comment:
form wrapper should have only a margin, height and width have to be on input
example:
```
.search-bar {
  display: flex;
  justify-content: center;
  margin: 20px auto 0;
}

.input {
  width: 100%;
  height: 70px;
  //other similar styles for both sizes
}

.input--small {
  height: 42px;
  //other difference styles
}
```